2017-02-10 22:18:01,092 p=1601 u=gregdonovan |   
2017-02-10 22:18:01,093 p=1601 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible-playbook addISSAusers -i inventory
2017-02-10 22:18:01,093 p=1601 u=gregdonovan |   
2017-02-10 22:18:01,094 p=1601 u=gregdonovan |  ERROR: the playbook: addISSAusers does not appear to be a file
2017-02-10 22:19:28,992 p=1611 u=gregdonovan |   
2017-02-10 22:19:28,992 p=1611 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible-playbook addISSAusers.yml -i inventory
2017-02-10 22:19:28,993 p=1611 u=gregdonovan |   
2017-02-10 22:19:28,994 p=1611 u=gregdonovan |  ERROR: the playbook: addISSAusers.yml could not be found
2017-02-10 22:23:46,924 p=1645 u=gregdonovan |   
2017-02-10 22:23:46,924 p=1645 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible-playbook addISSAusers.yml -i inventory
2017-02-10 22:23:46,924 p=1645 u=gregdonovan |   
2017-02-10 22:23:46,941 p=1645 u=gregdonovan |  ERROR: cannot find role in /Users/gregdonovan/ansible/roles/addISSAusers--- or /Users/gregdonovan/ansible/addISSAusers--- or /usr/local/etc/ansible/roles/addISSAusers---
2017-04-20 12:25:10,619 p=2733 u=gregdonovan |  
2017-04-20 12:25:10,621 p=2733 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible --version
2017-04-20 12:25:10,621 p=2733 u=gregdonovan |  
2017-04-20 12:25:41,385 p=2734 u=gregdonovan |  
2017-04-20 12:25:41,385 p=2734 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible --version
2017-04-20 12:25:41,385 p=2734 u=gregdonovan |  
2017-04-20 13:15:42,984 p=2965 u=gregdonovan |  
2017-04-20 13:15:42,985 p=2965 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible all -m ping
2017-04-20 13:15:42,985 p=2965 u=gregdonovan |  
2017-04-20 13:15:42,987 p=2965 u=gregdonovan |  ERROR: Unable to find an inventory file, specify one with -i ?
2017-04-20 13:16:10,990 p=2968 u=gregdonovan |  
2017-04-20 13:16:10,990 p=2968 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible all -i inventory -m ping
2017-04-20 13:16:10,990 p=2968 u=gregdonovan |  
2017-04-20 13:16:16,318 p=2968 u=gregdonovan |  beef | FAILED => SSH Error: ssh: connect to host beef port 22: Operation timed out
    while connecting to 192.168.1.150:22
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.
2017-04-20 13:16:16,321 p=2968 u=gregdonovan |  webz | FAILED => SSH Error: ssh: connect to host webz port 22: Operation timed out
    while connecting to 192.168.1.145:22
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.
2017-04-20 13:16:16,322 p=2968 u=gregdonovan |  think | FAILED => SSH Error: ssh: connect to host think port 22: Operation timed out
    while connecting to 192.168.1.147:22
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.
2017-04-20 13:16:16,330 p=2968 u=gregdonovan |  pi1 | FAILED => SSH Error: ssh: connect to host pi1 port 22: Operation timed out
    while connecting to 192.168.1.146:22
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.
2017-04-20 13:16:16,331 p=2968 u=gregdonovan |  webx | FAILED => SSH Error: ssh: connect to host webx port 22: Operation timed out
    while connecting to 192.168.1.141:22
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.
2017-04-20 13:16:16,332 p=2968 u=gregdonovan |  pi2 | FAILED => SSH Error: ssh: connect to host pi2 port 22: Operation timed out
    while connecting to 192.168.1.161:22
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.
2017-04-20 18:25:49,571 p=3421 u=gregdonovan |   
2017-04-20 18:25:49,572 p=3421 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible-playbook changepw -C -i inventory
2017-04-20 18:25:49,573 p=3421 u=gregdonovan |   
2017-04-20 18:25:49,574 p=3421 u=gregdonovan |  ERROR: the playbook: changepw could not be found
2017-04-20 18:26:03,935 p=3422 u=gregdonovan |   
2017-04-20 18:26:03,935 p=3422 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible-playbook changepw.yml -C -i inventory
2017-04-20 18:26:03,935 p=3422 u=gregdonovan |   
2017-04-20 18:26:03,952 p=3422 u=gregdonovan |  ERROR: found role at /Users/gregdonovan/ansible/roles/changepw, but cannot find /Users/gregdonovan/ansible/roles/changepw/tasks/main or /Users/gregdonovan/ansible/roles/changepw/handlers/main or /Users/gregdonovan/ansible/roles/changepw/vars/main or /Users/gregdonovan/ansible/roles/changepw/defaults/main or /Users/gregdonovan/ansible/roles/changepw/meta/main or /Users/gregdonovan/ansible/roles/changepw/library
2017-04-20 18:27:23,589 p=3431 u=gregdonovan |   
2017-04-20 18:27:23,589 p=3431 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible-playbook changepw.yml -C -i inventory
2017-04-20 18:27:23,589 p=3431 u=gregdonovan |   
2017-04-20 18:27:23,598 p=3431 u=gregdonovan |  ERROR: found role at /Users/gregdonovan/ansible/roles/changepw, but cannot find /Users/gregdonovan/ansible/roles/changepw/tasks/main or /Users/gregdonovan/ansible/roles/changepw/handlers/main or /Users/gregdonovan/ansible/roles/changepw/vars/main or /Users/gregdonovan/ansible/roles/changepw/defaults/main or /Users/gregdonovan/ansible/roles/changepw/meta/main or /Users/gregdonovan/ansible/roles/changepw/library
2017-04-20 18:29:51,789 p=3440 u=gregdonovan |   
2017-04-20 18:29:51,789 p=3440 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible-playbook changepw.yml -C -i inventory
2017-04-20 18:29:51,789 p=3440 u=gregdonovan |   
2017-04-20 18:29:51,856 p=3440 u=gregdonovan |  PLAY [temp] ******************************************************************* 
2017-04-20 18:29:51,857 p=3440 u=gregdonovan |  GATHERING FACTS *************************************************************** 
2017-04-20 18:29:54,532 p=3440 u=gregdonovan |  ok: [beef]
2017-04-20 18:29:54,683 p=3440 u=gregdonovan |  ok: [webz]
2017-04-20 18:29:55,397 p=3440 u=gregdonovan |  ok: [webx]
2017-04-20 18:29:57,197 p=3440 u=gregdonovan |  ok: [pi1]
2017-04-20 18:29:57,209 p=3440 u=gregdonovan |  TASK: [changepw | change user password] *************************************** 
2017-04-20 18:29:57,696 p=3440 u=gregdonovan |  changed: [webz]
2017-04-20 18:29:57,765 p=3440 u=gregdonovan |  changed: [beef]
2017-04-20 18:29:57,770 p=3440 u=gregdonovan |  changed: [webx]
2017-04-20 18:29:59,332 p=3440 u=gregdonovan |  changed: [pi1]
2017-04-20 18:29:59,342 p=3440 u=gregdonovan |  PLAY RECAP ******************************************************************** 
2017-04-20 18:29:59,342 p=3440 u=gregdonovan |  beef                       : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:29:59,343 p=3440 u=gregdonovan |  pi1                        : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:29:59,343 p=3440 u=gregdonovan |  webx                       : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:29:59,343 p=3440 u=gregdonovan |  webz                       : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:30:35,504 p=3501 u=gregdonovan |   
2017-04-20 18:30:35,504 p=3501 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible-playbook changepw.yml -C -i inventory
2017-04-20 18:30:35,504 p=3501 u=gregdonovan |   
2017-04-20 18:30:35,542 p=3501 u=gregdonovan |  PLAY [temp] ******************************************************************* 
2017-04-20 18:30:35,543 p=3501 u=gregdonovan |  GATHERING FACTS *************************************************************** 
2017-04-20 18:30:36,507 p=3501 u=gregdonovan |  ok: [webz]
2017-04-20 18:30:36,508 p=3501 u=gregdonovan |  ok: [beef]
2017-04-20 18:30:36,600 p=3501 u=gregdonovan |  ok: [webx]
2017-04-20 18:30:38,656 p=3501 u=gregdonovan |  ok: [pi1]
2017-04-20 18:30:40,661 p=3501 u=gregdonovan |  fatal: [pi2] => SSH Error: ssh: connect to host pi2 port 22: Operation timed out
    while connecting to 192.168.1.161:22
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.
2017-04-20 18:30:40,674 p=3501 u=gregdonovan |  TASK: [changepw | change user password] *************************************** 
2017-04-20 18:30:41,123 p=3501 u=gregdonovan |  changed: [webz]
2017-04-20 18:30:41,172 p=3501 u=gregdonovan |  changed: [beef]
2017-04-20 18:30:41,175 p=3501 u=gregdonovan |  changed: [webx]
2017-04-20 18:30:42,391 p=3501 u=gregdonovan |  changed: [pi1]
2017-04-20 18:30:42,401 p=3501 u=gregdonovan |  PLAY RECAP ******************************************************************** 
2017-04-20 18:30:42,402 p=3501 u=gregdonovan |             to retry, use: --limit @/Users/gregdonovan/changepw.retry

2017-04-20 18:30:42,402 p=3501 u=gregdonovan |  beef                       : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:30:42,402 p=3501 u=gregdonovan |  pi1                        : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:30:42,402 p=3501 u=gregdonovan |  pi2                        : ok=0    changed=0    unreachable=1    failed=0   
2017-04-20 18:30:42,402 p=3501 u=gregdonovan |  webx                       : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:30:42,402 p=3501 u=gregdonovan |  webz                       : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:40:43,039 p=3561 u=gregdonovan |   
2017-04-20 18:40:43,039 p=3561 u=gregdonovan |  /usr/local/Cellar/ansible/1.9.4/libexec/bin/ansible-playbook changepw.yml -i inventory
2017-04-20 18:40:43,039 p=3561 u=gregdonovan |   
2017-04-20 18:40:43,078 p=3561 u=gregdonovan |  PLAY [temp] ******************************************************************* 
2017-04-20 18:40:43,079 p=3561 u=gregdonovan |  GATHERING FACTS *************************************************************** 
2017-04-20 18:40:45,000 p=3561 u=gregdonovan |  ok: [webz]
2017-04-20 18:40:45,001 p=3561 u=gregdonovan |  ok: [beef]
2017-04-20 18:40:45,212 p=3561 u=gregdonovan |  ok: [webx]
2017-04-20 18:40:47,834 p=3561 u=gregdonovan |  ok: [pi1]
2017-04-20 18:40:48,162 p=3561 u=gregdonovan |  fatal: [pi2] => SSH Error: ssh: connect to host pi2 port 22: Operation timed out
    while connecting to 192.168.1.161:22
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.
2017-04-20 18:40:48,174 p=3561 u=gregdonovan |  TASK: [changepw | change user password] *************************************** 
2017-04-20 18:40:48,721 p=3561 u=gregdonovan |  changed: [beef]
2017-04-20 18:40:48,860 p=3561 u=gregdonovan |  changed: [webz]
2017-04-20 18:40:48,970 p=3561 u=gregdonovan |  changed: [webx]
2017-04-20 18:40:50,595 p=3561 u=gregdonovan |  changed: [pi1]
2017-04-20 18:40:50,608 p=3561 u=gregdonovan |  PLAY RECAP ******************************************************************** 
2017-04-20 18:40:50,608 p=3561 u=gregdonovan |             to retry, use: --limit @/Users/gregdonovan/changepw.retry

2017-04-20 18:40:50,609 p=3561 u=gregdonovan |  beef                       : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:40:50,609 p=3561 u=gregdonovan |  pi1                        : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:40:50,610 p=3561 u=gregdonovan |  pi2                        : ok=0    changed=0    unreachable=1    failed=0   
2017-04-20 18:40:50,610 p=3561 u=gregdonovan |  webx                       : ok=2    changed=1    unreachable=0    failed=0   
2017-04-20 18:40:50,610 p=3561 u=gregdonovan |  webz                       : ok=2    changed=1    unreachable=0    failed=0   
